components/panther.rst

```diff
@@ -0,0 +1,827 @@
+``PANTHER_ERROR_SCREENSHOT_DIR``
+    Set a base directory for your failure/error screenshots (e.g. ``./var/error-screenshots``)
+``PANTHER_DEVTOOLS``
+    Toggle the browser's dev tools (default ``enabled``, useful to debug)
+``PANTHER_ERROR_SCREENSHOT_ATTACH``
+    Add screenshots mentioned above to test output in junit attachment format
+
+Chrome Specific Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``PANTHER_NO_SANDBOX``
+    Disable `Chrome's sandboxing`_ (unsafe, but allows to use Panther in containers)
+``PANTHER_CHROME_ARGUMENTS``
+    Customize Chrome arguments. You need to set ``PANTHER_NO_HEADLESS`` to fully customize
+``PANTHER_CHROME_BINARY``
+    To use another ``google-chrome`` binary
+
+Firefox Specific Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``PANTHER_FIREFOX_ARGUMENTS``
+    Customize Firefox arguments. You need to set ``PANTHER_NO_HEADLESS`` to fully customize
+``PANTHER_FIREFOX_BINARY``
+    To use another ``firefox`` binary
+
+.. _panther_interactive-mode:
+
+Interactive Mode
+----------------
+
+Panther can make a pause in your tests suites after a failure.
+Thanks to this break time, you can investigate the encountered problem through
+the web browser. To enable this mode, you need the ``--debug`` PHPUnit option
+without the headless mode:
+
+.. code-block:: terminal
+
+    $ PANTHER_NO_HEADLESS=1 bin/phpunit --debug
+
+    Test 'App\AdminTest::testLogin' started
+    Error: something is wrong.
+
+    Press enter to continue...
+
+To use the interactive mode, the
+:ref:`PHPUnit extension <panther_phpunit-extension>` has to be registered.
+
+Docker Integration
+------------------
+
+Here is a minimal Docker image that can run Panther with both Chrome and
+Firefox:
+
+.. code-block:: dockerfile
+
+    FROM php:alpine
+
+    # Chromium and ChromeDriver
+    ENV PANTHER_NO_SANDBOX 1
+    # Not mandatory, but recommended
+    ENV PANTHER_CHROME_ARGUMENTS='--disable-dev-shm-usage'
+    RUN apk add --no-cache chromium chromium-chromedriver
+
+    # Firefox and GeckoDriver (optional)
+    ARG GECKODRIVER_VERSION=0.28.0
+    RUN apk add --no-cache firefox libzip-dev; \
+        docker-php-ext-install zip
+    RUN wget -q https://github.com/mozilla/geckodriver/releases/download/v$GECKODRIVER_VERSION/geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz; \
+        tar -zxf geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz -C /usr/bin; \
+        rm geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz
+
+You can then build and run your image:
+
+.. code-block:: bash
+
+    $ docker build . -t myproject
+    $ docker run -it -v "$PWD":/srv/myproject -w /srv/myproject myproject bin/phpunit
+
+Integrating Panther In Your CI
+------------------------------
+
+Github Actions
+~~~~~~~~~~~~~~
+
+Panther works out of the box with `GitHub Actions`_.
+Here is a minimal ``.github/workflows/panther.yaml`` file to run Panther tests:
+
+.. code-block:: yaml
```

Comment:
@javiereguiluz WDYT ich we use a TAB directive here for the different CI systems?